fe/fe-core/src/main/java/org/apache/doris/nereids/hint/LeadingHint.java

```diff
@@ -401,17 +482,6 @@ public Plan generateLeadingJoinPlan() {
 
         LogicalJoin finalJoin = (LogicalJoin) stack.pop().second;
         // we want all filters been remove
-        if (!getFilters().isEmpty()) {
```

Comment:
Pls add a checking for getFilters is not empty.

Reply:
done